AnalysisTools/python/templates/MuonBaseFlow.py

```diff
@@ -27,4 +27,15 @@ def addGhostCleaning(self, step):
                              passthrough = cms.string("isGlobalMuon && numberOfMatches >= 2"),
                              fractionOfSharedSegments = cms.double(0.499))
 
-        step.addModule("muonGhostCleaning", mod, 'm')            
+        step.addModule("muonGhostCleaning", mod, 'm')
+    def addMuonPOGIDs(self, step):
+        '''
+        Add Muon POG IDs as UserInts
+
+        '''
+        embedMuId = cms.EDProducer(
+                "MuonIdEmbedder",
+                src = step.getObjTag('m'),
+                vertexSrc = cms.InputTag('offlineSlimmedPrimaryVertices')
```

Comment:
It probably doesn't matter here, but it's probably better to use `step.getObjTag('v')` here.